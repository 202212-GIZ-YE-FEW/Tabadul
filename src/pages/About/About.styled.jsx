import styled from "@emotion/styled";
import Image from "next/image";

export const AboutSection = styled.div`
    background-color: #fff;
`;
export const Container = styled.div`
    margin-left: auto;
    margin-right: auto;
    padding-left: 15px;
    padding-right: 15px;
    @media (min-width: 768px) {
        width: 750px;
    }
    /* Medium */
    @media (min-width: 992px) {
        width: 970px;
    }
    @media (min-width: 1200px) {
        width: 1170px;
    }
    /* Large */
    @media (min-width: 1500px) {
        width: 1500px;
    }
`;
export const HeroSection = styled.div`
    height: 100vh;
    display: flex;
    flex-wrap: wrap;
    align-items: center;
    justify-content: center;
    @media screen and (max-width: 991px) {
        height: auto;
        flex-direction: column-reverse;
    }
`;
export const TextContent = styled.div`
    flex: 1;
`;
export const ImgContent = styled.div`
    flex: 1;
`;
export const AboutImg = styled(Image)`
    max-width: 100%;
    @media screen and (max-width: 768px) {
        height: 300px;
    }
`;
export const HeadingContent = styled.h3`
    font-style: normal;
    font-weight: 800;
    font-size: 36px;
    line-height: 54px;
    color: #3c4347;
    @media screen and (max-width: 768px) {
        font-size: 24px;
        line-height: 20px;
        text-align: center;
    }
`;
export const ParagrhContent = styled.p`
    font-style: normal;
    font-weight: 400;
    font-size: 24px;
    line-height: 36px;
    width: 720px;
    color: #3c4347;
    @media (min-width: 1200px) and (max-width: 1500px) {
        width: 500px;
        font-size: 20px;
        line-height: 30px;
    }
    @media (min-width: 768px) and (max-width: 1200px) {
        width: fit-content;
    }
    @media screen and (max-width: 768px) {
        width: fit-content;
        font-weight: 400;
        font-size: 20px;
        line-height: 30px;
        text-align: center;
    }
`;

export const TeamSection = styled.div`
    padding-top: 75px;
    padding-bottom: 75px;
    @media screen and (max-width: 768px) {
        padding-top: 50px;
        padding-bottom: 50px;
    }
`;
export const TeamHeading = styled.h3`
    font-style: normal;
    font-weight: 800;
    font-size: 36px;
    line-height: 54px;
    text-align: center;
    color: #3c4347;
    @media screen and (max-width: 768px) {
        font-size: 24px;
        line-height: 20px;
        text-align: center;
        margin-bottom: 50px;
        font-weight: 700;
    }
`;
export const TeamMemberContainer = styled.div`
    display: flex;
    align-items: center;
    justify-content: space-between;
    flex-wrap: wrap;
    gap: 45px;
`;
