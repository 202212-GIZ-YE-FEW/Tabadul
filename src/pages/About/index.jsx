import React from "react";

import {
    AboutImg,
    AboutSection,
    Container,
    HeadingContent,
    HeroSection,
    ImgContent,
    ParagrhContent,
    TeamHeading,
    TeamMemberContainer,
    TeamSection,
    TextContent,
} from "./About.styled";
import AboutPhoto from "../../assets/Image/About.svg";
import { teams } from "../../../data/db";
function About() {
    return (
        <AboutSection>
            <Container>
                <HeroSection>
                    <TextContent>
                        <HeadingContent>Who we are</HeadingContent>
                        <ParagrhContent>
                            Lorem ipsum dolor sit amet, consectetur adipiscing
                            elit, sed do eiusmod tempor incididunt ut labore et
                            dolore magna aliqua. Ut enim ad minim veniam, quis
                            nostrud exercitation ullamco laboris nisi ut aliquip
                            ex ea commodo consequat. Ut enim ad minim veniam,
                            quis nostrud exercitation ullamco laboris nisi ut
                            aliquip ex ea commodo consequat.
                        </ParagrhContent>
                    </TextContent>
                    <ImgContent>
                        <AboutImg
                            src={AboutPhoto}
                            alt='AboutPhoto'
                            width={896}
                            height={651}
                            priority={true}
                        />
                    </ImgContent>
                </HeroSection>
                <TeamSection>
                    <TeamHeading>Our Team</TeamHeading>
                    <TeamMemberContainer>
                        {teams?.map((member) => {
                            return <h3 key={member.id}>{member.Surename}</h3>;
                        })}
                    </TeamMemberContainer>
                </TeamSection>
            </Container>
        </AboutSection>
    );
}

export default About;
