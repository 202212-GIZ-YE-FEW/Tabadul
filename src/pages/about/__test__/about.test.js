import renderer from "react-test-renderer";

import About from "../index.jsx";
it("renders correctly", () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
});
