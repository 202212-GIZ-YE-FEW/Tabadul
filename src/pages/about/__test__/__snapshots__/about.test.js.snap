// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-15tqd4u"
>
  <div
    className="css-w5ihqc"
  >
    <div
      className="css-1h3rc7l"
    >
      <div
        className="css-17ugp5u"
      >
        <h3
          className="css-113i9g4"
        >
          Who we are
        </h3>
        <p
          className="css-1pklm73"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
        </p>
      </div>
      <div
        className="css-eie7o"
      >
        <img
          alt="AboutPhoto"
          className="css-1n19hu5"
          data-nimg="1"
          decoding="async"
          height={651}
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2Fimg.jpg&w=1920&q=75"
          srcSet="/_next/image?url=%2Fimg.jpg&w=1080&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1920&q=75 2x"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={896}
        />
      </div>
    </div>
    <div
      className="css-cl3gno"
    >
      <h3
        className="css-1mvkmlg"
      >
        Our Team
      </h3>
      <div
        className="css-jmcdrs"
      >
        <div
          className="css-wr9924"
        >
          <img
            alt="TeamImg"
            className="css-ksyvgr"
            data-nimg="1"
            decoding="async"
            height={40}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={40}
          />
          <h3
            className="css-1c2ehun"
          >
            Name Surename
          </h3>
          <p
            className="css-1md5ex3"
          >
            Description for member
          </p>
        </div>
        <div
          className="css-wr9924"
        >
          <img
            alt="TeamImg"
            className="css-ksyvgr"
            data-nimg="1"
            decoding="async"
            height={40}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={40}
          />
          <h3
            className="css-1c2ehun"
          >
            Name Surename
          </h3>
          <p
            className="css-1md5ex3"
          >
            Description for member
          </p>
        </div>
        <div
          className="css-wr9924"
        >
          <img
            alt="TeamImg"
            className="css-ksyvgr"
            data-nimg="1"
            decoding="async"
            height={40}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={40}
          />
          <h3
            className="css-1c2ehun"
          >
            Name Surename
          </h3>
          <p
            className="css-1md5ex3"
          >
            Description for member
          </p>
        </div>
        <div
          className="css-wr9924"
        >
          <img
            alt="TeamImg"
            className="css-ksyvgr"
            data-nimg="1"
            decoding="async"
            height={40}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={40}
          />
          <h3
            className="css-1c2ehun"
          >
            Name Surename
          </h3>
          <p
            className="css-1md5ex3"
          >
            Description for member
          </p>
        </div>
        <div
          className="css-wr9924"
        >
          <img
            alt="TeamImg"
            className="css-ksyvgr"
            data-nimg="1"
            decoding="async"
            height={40}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={40}
          />
          <h3
            className="css-1c2ehun"
          >
            Name Surename
          </h3>
          <p
            className="css-1md5ex3"
          >
            Description for member
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
